packs/data/extinction-curse-bestiary.db/shoony-hierarch.json

```diff
@@ -174,7 +433,7 @@
             },
             "img": "systems/pf2e/icons/default-icons/equipment.svg",
             "name": "longbow",
```

Comment:
Delete

Reply:
Why are we wanting to delete the equipment from the Hierarch and Militia Member? It's in the stat block.

Reply:
Fixed in [5f3c718](https://github.com/foundryvtt/pf2e/pull/2810/commits/5f3c718c6b78d33f4c335a7b9398895043044537)